### PR TITLE
Added a journey page

### LINKED ISSUE #100: Add a team page
This will have a journey design, starting from the first person to the recent changes in the team with their responsibilities. I wish to take it up but after some time and I already have to work on enabling search on the website[high prio than this]. If someone is interested to work on this please contact me on Slack channel. 

Happy to have design suggestions for this page which goes with the theme of our main website.

### PR DESCRIPTION
**Pull request by anubhakushwaha**
**Checklist**

- [x] My branch is up-to-date with the upstream `predev` branch.
- [x] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #100 

If relevant, please include a screenshot.
screenshot from 2017-12-18 23-14-40

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
@tapasweni-pathak  @vaibhavsingh97  @jarifibrahim  @nikhita  Please review and suggestions on improving it would be great, thanks

File: oshc/main/models.py [hunk 1]
```diff
@@ -29,3 +29,12 @@ class Contest(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Journey(models.Model):
+    title = models.CharField(max_length=128, help_text="Journey title")
+    start_date = models.DateField(null=True)
+
```
Comment (jarifibrahim) [inline oshc/main/models.py:37]: Can we add a `description` field too?

File: oshc/main/views.py [hunk 2]
```diff
@@ -38,3 +38,9 @@ def add_contest(request):
 
 def submit_contest(request):
     return render(request, 'contest_submission.html')
+
+
+def journey(request):
+    journey_list = Journey.objects.all()
+    return render(request, 'journey.html',
+                  context={'Journey': journey_list})
```
Comment (jarifibrahim) [inline oshc/main/views.py:46]: You might want to sort the `journey_list` by the `start_date` before sending it to the HTML page. This ensures that the data we send out of the view is chronologically ordered.
You can use `Managers`[0] to achieve this or you could do something like `Journey.objects.all().filteryBy('start_date')` 

[0] - https://docs.djangoproject.com/en/2.0/topics/db/managers/

File: oshc/main/templates/base.html [hunk 3]
```diff
@@ -33,7 +33,8 @@
             </ul>
           </li>
           <li><a href="https://github.com/OpenSourceHelpCommunity" target="_blank">Resources</a></li>
-            <li><a href="{% url 'contests' %}" target="_blank">Contests</a></li>
+          <li><a href="{% url 'contests' %}" target="_blank">Contests</a></li>
+          <li><a href="{% url 'journey' %}" target="_blank">Journey</a></li>
```
Comment (jarifibrahim) [inline oshc/main/templates/base.html:37]: I don't think we should put `target="_blank"` here. The Journey page is part of the OSHC page.

**Comment by jarifibrahim** [comment]
@anubhakushwaha Is there any way we could make the timeline responsive?

**Comment by anubhakushwaha** [comment]
@jarifibrahim Hey thanks for reviewing will do the changes, responsive as in? The current behavior allows new events to be added via the admin. Please help me understand this better.

**Comment by jarifibrahim** [comment]
@anubhakushwaha By responsive I meant is the timeline mobile device friendly? As far as I know, the OSHC website is mobile friendly, so it would be great if we could make the timeline mobile friendly as well.

**Comment by vaibhavsingh97** [comment]
@anubhakushwaha Can we have live demo link please

**Comment by anubhakushwaha** [comment]
@vaibhavsingh97  Live demo for?

File: oshc/main/models.py (hunk 1 above)
Comment (tapaswenipathak) [reply oshc/main/models.py:37]: We are good w/ these two.

File: oshc/main/templates/base.html (hunk 3 above)
Comment (tapaswenipathak) [reply oshc/main/templates/base.html:37]: These needs to be removed for quite a few.

File: oshc/main/views.py (hunk 2 above)
Comment (tapaswenipathak) [reply oshc/main/views.py:46]: fixed w/ `Journey.objects.order_by('start_date')`.

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]
Thanks for sending this through @anubhakushwaha! 🎉 

I have fixed few minor reviews that you have got till now. We need to make this work even on a mobile sized/small screens.

**Comment by tapaswenipathak** [comment]
This is merged after few fixes w/ 7922feea6bb485b3d09c6639d372f82e05615e14, live [here](http://opensourcehelpcommunity-predev.herokuapp.com/journey/).

**Comment by anubhakushwaha** [comment]
@tapasweni-pathak  Thank you for doing the changes. 
> We need to make this work even on a mobile sized/small screens.

I will update you on this.

**Pull request closed without merging**